### PR TITLE
Update spanish translations

### PR DESCRIPTION
**Pull request by lesthack**
I translate the new labels, and shorted some words because in spanish are very largest and overlap.

Before:
screenshot from 2018-10-24 11-38-04

After:
screenshot from 2018-10-24 11-39-51

### CODE REVIEW

File: src/_locales/es-ES.json [hunk 1]
```diff
@@ -131,22 +131,22 @@
...
   "title-settings-playback": "Reproducción",
   "title-settings-general": "General",
-  "title-settings-hotkeys": "Atajos de Teclado",
+  "title-settings-hotkeys": "Atajos",
   "title-settings-lastfm": "LastFM",
   "title-settings-listenbrainz": "ListenBrainz",
-  "title-settings-mini": "Reproductor Mini",
-  "title-settings-style": "Estilos Personalizados",
-  "title-settings-slack": "",
+  "title-settings-mini": "Mini",
```
Comment (waldyrious) [inline src/_locales/es-ES.json]: Would "Mini Player" work in Spanish? In Portuguese it's common to use the English word if the native translation is awkward.

File: src/_locales/es-ES.json [hunk 2]
```diff
@@ -131,22 +131,22 @@
   "settings-options-style-dialog-button": "Cargar CSS",
   "settings-options-style-dialog-css-files": "Ficheros CSS",
   "settings-options-style-dialog-all-files": "Todos los Ficheros",
-  "slack-token-label": "",
-  "slack-token-placeholder": "",
+  "slack-token-label": "Ingresa tu token de usuario slack (puedes encontrarlo en https://api.slack.com/custom-integrations/legacy-tokens)",
```
Comment (waldyrious) [inline src/_locales/es-ES.json]:
```suggestion
  "slack-token-label": "Ingresa tu token de usuario Slack (puedes encontrarlo en https://api.slack.com/custom-integrations/legacy-tokens)",
```

Comment (lesthack) [reply src/_locales/es-ES.json]: Sure, thank you.

File: src/_locales/es-ES.json (hunk 1 above)
Comment (lesthack) [reply src/_locales/es-ES.json]: Yes, in spanish also. So, I think that we can change this without problem.

File: src/_locales/es-ES.json [hunk 3]
```diff
@@ -131,22 +131,22 @@
...
   "title-settings-playback": "Reproducción",
   "title-settings-general": "General",
-  "title-settings-hotkeys": "Atajos de Teclado",
+  "title-settings-hotkeys": "Atajos",
   "title-settings-lastfm": "LastFM",
   "title-settings-listenbrainz": "ListenBrainz",
-  "title-settings-mini": "Reproductor Mini",
-  "title-settings-style": "Estilos Personalizados",
-  "title-settings-slack": "",
+  "title-settings-mini": "Mini",
```
Comment (waldyrious) [inline src/_locales/es-ES.json]:
```suggestion
  "title-settings-mini": "Mini Player",
```

**Pull request merged**